app/models/concerns/flags.rb

```diff
@@ -1,5 +1,6 @@
 module Flags
   extend ActiveSupport::Concern
+  FLAGS = %w(production, archived, alpha, beta)
 
```

Comment:
@marcellosachs You can remove this concern, it duplicates what is already in the `lib/activity_flags.rb` file.